bin/markcop

```diff
@@ -25,6 +26,80 @@ errors=''
 IFS='
 '
 
+# Check for trailing whitespace
+printf '\n\nChecking for trailing whitespace...\n'
+
+for line in $(git ls-files | grep '\.md$' | xargs grep -noP '^.+\s$' | sed -e 's/:/\t/' -e 's/:/\t/'); do
```

Comment:
See line #16 of https://github.com/hackedu/meta/blob/master/markdown_style_guide.md